integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2920,6 +2920,15 @@ msgstr ""
 msgid "import errors"
 msgstr "Importfehler"
 
+#: cms/models/external_calendars/external_calendar.py
+#: cms/templates/events/external_calendar_list.html
+msgid "Created by"
+msgstr "Erstellt von"
```

Comment:
should we use the same words that we use for abstract content translations (creator and Ersteller:in)?